core/lexicon/en/file.inc.php

```diff
@@ -73,6 +73,7 @@
 $_lang['file_folder_remove'] = 'Delete Directory';
 $_lang['file_folder_remove_confirm'] = 'Are you sure you want to delete the directory: "[[+directory]]"?<br />This could potentially break your website.';
 $_lang['file_folder_rename'] = 'Rename Directory';
+$_lang['file_info_metadata_new'] = "Unable to access metaData for the file “[[+file]]” because it is for a new element and has not been created yet. An attempt will be made to create the file automatically.";
```

Comment:
You are using the wrong quotes. I don't think it's important, but it's better to keep them in the same style (since crowdin sometimes behaves strangely) :)

Reply:
Yeah, I'm picky about using single quotes everywhere possible ... missed that here, will change! On the other quotes, they are proper quotation marks for text that I've used elsewhere (using the utf-8 curly quotation marks instead of html entities).

Reply:
It is better to lead to one style.

Reply:
Made changes to the lex file. Should be all consistent now.